src/behaviours/update-app/installing-update-using-tray.test.ts

```diff
@@ -38,9 +43,12 @@ describe("installing update using tray", () => {
       downloadPlatformUpdateMock = asyncFn();
       setUpdateOnQuitMock = jest.fn();
       showApplicationWindowMock = jest.fn();
+      showNotificationMock = jest.fn();
+      askBooleanMock = asyncFn();
 
+      mainDi.override(showNotificationInjectable, () => showNotificationMock);
```

Comment:
MainDi should have no concept of notifications at all.

Reply:
This is removed in latter commit.